src/services/ApiService.ts

```diff
@@ -183,16 +219,30 @@ export class ApiService {
         this.clearToken();
         if (ApiService.RuntimeConfig.googleClientId) {
             this.authInstance?.signOut();
+            // Redirect to dashboard URL if it exists
+            if (ApiService.RuntimeConfig.dashboardAddress) {
+                window.open(ApiService.RuntimeConfig.dashboardAddress, "_self");
+            }
+        } else if (ApiService.RuntimeConfig.oidcClientId) {
+            let usp = new URLSearchParams();
+            usp.set("id_token_hint", localStorage.getItem("oidc_id_token"));
+            usp.set("post_logout_redirect_uri", window.location.href);
+
+            localStorage.removeItem("oidc_refresh_token");
+            localStorage.removeItem("oidc_id_token");
+            localStorage.removeItem("authenticationType");
+
+            window.open(ApiService.RuntimeConfig.logoutAddress + "?" + usp.toString(), "_self");
```

Comment:
I'd prefer `${ApiService.RuntimeConfig.logoutAddress}?${usp.toString()}` but I don't have a strong opinion about it :slightly_smiling_face: